package-scripts.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     },
     test: {
       default: 'cross-env NODE_ENV=test jest --coverage',
+      update: 'cross-env NODE_ENV=test jest -u',
```

Comment:
I'll try to work on that feature I was talking about and then that should totally work